src/components/PageWithSideMenu.tsx

```diff
@@ -75,34 +73,34 @@ interface LinkButtonProps extends ButtonProps, Pick<LinkProps, 'to'> {
 }
 
 export const LinkButton = ({ isSelected, ...rest }: LinkButtonProps) => (
-  <li>
-    <Button variant={isSelected ? 'contained' : 'outlined'} size="large" LinkComponent={Link} {...rest} />
-  </li>
+  <Button
+    sx={{ bgcolor: isSelected ? 'primary.main' : 'white', ml: '1rem', width: 'min-content' }}
```

Comment:
Kolliderer ikke dette med `variant`-sjekken under?
```suggestion
    sx={{ ml: '1rem', width: 'min-content' }}
```

Reply:
Finner ingen åpenbar konflikt mellom de to. Fargen endrer seg, og button veksler mellom contained og outlined.